spec/swagger_schemas/models/enrolment_csv.rb

```diff
@@ -50,6 +50,12 @@
         type: :boolean,
         example: true,
       },
+      funded_place: {
+        description: "Indicates whether this NPQ participant is funded by DfE",
+        type: :boolean,
+        example: true,
+        required: true,
+      },
```

Comment:
Can we move this down below `school_urn` to maintain the same ordering as the serializer.

Reply:
Updated 👍